lib/metadataTypes/TriggeredSendSummary.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * MessageSendActivity MetadataType
+ *
+ * @augments MetadataType
+ */
+class TriggeredSendSummary extends MetadataType {
+    /**
+     * Retrieves SOAP based metadata of metadata type into local filesystem. executes callback with retrieved metadata
+     *
+     * @param {string} retrieveDir Directory where retrieved metadata directory will be saved
+     * @param {void | string[]} [_] unused parameter
+     * @param {void | string[]} [__] unused parameter
+     * @param {string} [key] customer key of single item to retrieve
+     * @returns {Promise.<MetadataTypeMapObj>} Promise of metadata
+     */
+    static retrieve(retrieveDir, _, __, key) {
+        /** @type {SoapRequestParams} */
+        let requestParams = {};
+        if (key) {
+            // move original filter down one level into rightOperand and add key filter into leftOperand
+            requestParams = {
+                filter: {
+                    leftOperand: {
+                        leftOperand: 'CustomerKey',
+                        operator: 'equals',
+                        rightOperand: key,
+                    },
+                    operator: 'AND',
+                    rightOperand: requestParams.filter,
+                },
+            };
+        }
+
+        return super.retrieveSOAP(retrieveDir, requestParams, key);
+    }
+    /**
+     * parses retrieved Metadata before saving
+     *
+     * @param {MetadataTypeItem} metadata a single item
+     * @returns {MetadataTypeItem | void} Array with one metadata object and one sql string
+     */
+    static postRetrieveTasks(metadata) {
+        try {
+            // CustomerKey for this type is the same as the CustomerKey of the TriggeredSend it relates to
+            const triggeredSendCategoryID = cache.searchForField(
+                'triggeredSend',
+                metadata.CustomerKey,
+                'CustomerKey',
+                'CategoryID'
+            );
+            metadata['TriggeredSend.CategoryID'] = triggeredSendCategoryID;
+            // folder path for a TriggeredSendSummary is the same as for the related TriggeredSend. CategoryID is not retrievable for this type, so use the TriggeredSend's field
+            this.setFolderPath(metadata);
+            const triggeredSendName = cache.searchForField(
+                'triggeredSend',
+                metadata.CustomerKey,
+                'CustomerKey',
+                'Name'
+            );
+            metadata.r__triggeredSend_name = triggeredSendName;
+            const triggeredSendKey = cache.searchForField(
+                'triggeredSend',
+                metadata.CustomerKey,
+                'CustomerKey',
+                'CustomerKey'
+            );
+            metadata.r__triggeredSend_key = triggeredSendKey;
+        } catch {
+            // the function will throw an error if the field is not found. We don't need to store the summary if the related TriggeredSend is not found
+            throw new Error(
```

Comment:
dont throw an error in postRetrieveTasks to skip the item but instead ensure the function returns null or undefined. That will then be caught one level up and the item will be recorded as "filtered".

<img width="968" height="319" alt="image" src="https://github.com/user-attachments/assets/6fa76028-d051-47d5-8e5a-45bae2ed38ac" />

any logs generated by postRetrieveTasks usually use Util.logger.warn or Util.logger.verbose()
verbose is only visible in the log-file while warn is used for cases you really need the user to see. My guess is, verbose is the better option here.